server/internal/usecase/interactor/model.go

```diff
@@ -69,6 +69,10 @@ func (i Model) Create(ctx context.Context, param interfaces.CreateModelParam, op
 			if err != nil {
 				return nil, err
 			}
+			meta, err := schema.New().NewID().Workspace(p.Workspace()).Project(p.ID()).Build()
```

Comment:
Do you always create metadata? Not all models use metadata, so it is fine to create it on a delayed basis (when fields are added to the metadata for the first time).

Reply:
Is it possible to add fields to the metadata without the metadata existing in the schema? Does an implementation exist?

Reply:
metadata is created on field creation for the first time